orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>upper case J in J_2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thank you.</p>